cmd/dao_tests/pubkey_test.go

```diff
@@ -1,191 +1,121 @@
 //go:build integration
 // +build integration
 
+// To override application configuration for integration tests, copy local.yaml into this directory.
+
 package main
 
 import (
 	"context"
-	"fmt"
-	"os"
 	"testing"
 
-	"github.com/RHEnVision/provisioning-backend/internal/config"
 	"github.com/RHEnVision/provisioning-backend/internal/dao"
 	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
-	"github.com/RHEnVision/provisioning-backend/internal/db"
-	"github.com/RHEnVision/provisioning-backend/internal/models"
 	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
```

Comment:
I feel this package is purely implemented and TestMain is superior tool for the job (except the Suite context), can you share briefly why have you decided to go this way?